BMS.cpp

```diff
@@ -0,0 +1,63 @@
+#include "BMS.h"
+
+BMS::BMS()
+{
+}
+
+bool BMS::ValidateMax(float value, float max)
```

Comment:
Nice use of lambdas in the `addParameter` below. Can you use [= capture](https://en.cppreference.com/w/cpp/language/lambda#Lambda_capture) for `ValidateMax` and `ValidateMin`?
Try it... maybe you can directly use them instead of going through the parameter?